### PR TITLE
Create team.md

### PR DESCRIPTION
**Pull request by lst97**
Update User Interface Enhancement in team.mr and content in readme.md

# Description
Other team detail need to be updated.

## Type of change

- [x] Documentation (update)

# Checklist:

- [x] I have made corresponding changes to the documentation
- [x] I have requested a review from ... on the Pull Request

### CODE REVIEW

**Comment by Ishrat420** [comment]
Hi Nelson, Please run formatter prettier so that it will pass the check

File: docs/projects/t2-2022/team.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+ - Product Lead: [to be added]
+
+ ### User Interface Enhancement
+
+ - Teams Channel: [User Interface Enhancement](https://teams.microsoft.com/_?tenantId=d02378ec-1688-46d5-8540-1c28b5f470f6#/school/conversations/Front%20End%20Migration?groupId=0e15669c-3f66-49aa-b023-640fe1dda2e0&threadId=19:40cc53f7f52d42cd8d15bddad593aa01@thread.tacv2&ctx=channel)
+ - Trello Board: [User Interface Enhancement](https://trello.com/b/pFPgCaIo/front-end-migration)
+ - Delivery Lead: Leo Luong
+ - Academic Mentor: Muna Al-Hawawreh
+ - Team Members: Nelson Lai, Ahmed Ali, Aldy Putra Shavira, Anthony Papoutsis, Tushar, Haris Sajjad, Kosta Constantinou
```
Comment (leocomsci) [inline docs/projects/t2-2022/team.md]: I adjusted some content here already. @Ishrat420 could you double check for us again if everything is good to merge

Comment (Ishrat420) [reply docs/projects/t2-2022/team.md]: Everything else look fine, just run `npm run format:check` so that it will pass the check

**Comment by lst97** [comment]
@Ishrat420 
I used the Prettier to have the file format, still not pass. Any instruction or idea?
image

Using Prettier 2.7.1

File: docs/projects/t2-2022/team.md (hunk 1 above)
Comment (leocomsci) [reply docs/projects/t2-2022/team.md]: Screen Shot 2022-08-04 at 2 16 51 am
When I try ti run `npm run format:check`, what in my terminal is look like the image I attached. Then I run `npx prettier --write .`

Comment (leocomsci) [reply docs/projects/t2-2022/team.md]: It all passes now

**Review by Ishrat420 — APPROVED** [review]

**Pull request merged**